### ISSUE TITLE
Tellurium parameters (not in AMBER)

### ISSUE THREAD

**Issue by bannanc** [open-forcefield-group/smirff99Frosst#30]
Tellurium doesn't currently exist in the AMBER universe. Its probably not a high priority right now, but Te does show up in the DrugBank database.

**Comment by davidlmobley** [comment]
LOL. Tellurium. Sheesh. In a drug? The things in drugs... 

I wonder if we should just prune this one out.

**Comment by bannanc** [comment]
DrugBank is also full of metals, which are common in drugs. This could just go with those as things we don't cover.

**Comment by davidlmobley** [comment]
Yes. Probably need to strip most of the metals for now. We want to cover "normal" organic chemistry for the time being.

**Comment by jchodera** [comment]
I've even been injected with a compound containing Technetium!

**Comment by bannanc** [comment]
Ok, We don't have a "pruned" drugbank. I have DrugBank_atyped which has parm@frosst atomtypes that I'm using to check missing parameters right now. 
I has metals, their atomtypes get assigned as the atomic symbol or as 'gg' (no type) in some cases. 

It seemed like the easiest set to look at for what "normal" chemistry we might be missing. I have a list in my notes of things we don't cover from this set either because they aren't normal or don't exist in AMBER. I'll organize it for public eyes when I get this pull request finished.
